addons/iap/tests/test_iap.py

```diff
@@ -6,5 +6,7 @@
 
 class TestIAP(TransactionCase):
     def test_get_account(self):
-        account = self.env["iap.account"].get("random_service_name")
+        service_name = 'random_service_name'
+        self.env['iap.service'].create({'technical_name': service_name})
```

Comment:
The runbot is red because of this line, all the fields of `iap.service` are required now.